animations/test/misc/expand_card_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:animations/src/misc/expand_card.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createExpandCardScreen() => MaterialApp(
+      home: ExpandCardDemo(),
+    );
+
+void main() {
+  testWidgets('size of ExpandCard changes on tap', (tester) async {
+    await tester.pumpWidget(createExpandCardScreen());
+
+    Size size = tester.getSize(find.byType(AnimatedContainer));
+
+    // size of Expand Card is 128 initially
+    expect(size.width, 128);
+
+    // tap at the center of the screen
+    await tester.tapAt(const Offset(400.0, 300.0));
+    await tester.pumpAndSettle(Duration(milliseconds: 300));
+
+    size = tester.getSize(find.byType(AnimatedContainer));
+    // size of Expand Card is 256, when card is selected
+    expect(size.width, 256);
+  });
+
+  testWidgets('assets images are loaded properly', (tester) async {
```

Comment:
I don't know that this test is needed. It seems to be checking the specific assets that have been used to build the AnimatedCrossFade, which isn't likely to be something that ever fails.